nnstreamer_example/custom_example_opencv/nnstreamer_customfilter_opencv_scaler.cc

```diff
@@ -0,0 +1,170 @@
+/**
+ * NNStreamer OpenCV Custom Filter Example: Scaler
+ * 
+ * Copyright (C) 2018 Sangjung Woo <sangjung.woo@samsung.com>
+ *
+ * LICENSE: LGPL-2.1
+ *
+ * @file  nnstreamer_customfilter_opencv_scaler.cc
+ * @date  22 Oct 2018
+ * @brief  OpenCV Custom NNStreamer Filter Example: Scaler
+ * @author  Sangjung Woo <sangjung.woo@samsung.com>
+ * @bug  No known bugs
```

Comment:
BTW, why we want to use @bug tag in this PR? what for?

Reply:
All of examples that we made have the `@bug` tag. So I just copied it.